build/verify-features.ts

```diff
@@ -1,45 +1,64 @@
 #!/usr/bin/env -S TS_NODE_COMPILER_OPTIONS='{"module":"es2020"}' node --loader ts-node/esm
 
-import {readdirSync} from 'node:fs';
+import {existsSync, readdirSync, readFileSync} from 'node:fs';
 
 import {getFeatures, getFeaturesMeta} from './readme-parser.js'; // Must import as `.js`
 
 const featuresDirContents = readdirSync('source/features/');
+const entryPoint = 'source/refined-github.ts';
+const entryPointSource = readFileSync(entryPoint);
 const importedFeatures = getFeatures();
 const featuresInReadme = getFeaturesMeta();
 
-const errors: string[] = [];
+function findError(filename: string): string | void {
+	if (filename === 'index.tsx') {
+		return;
+	}
+
+	if (filename.endsWith('.css')) {
+		const correspondingTsxFile = `source/features/${filename.replace(/.css$/, '.tsx')}`;
+		if (existsSync(correspondingTsxFile)) {
+			if (!readFileSync(correspondingTsxFile).includes(`import './${filename}';`)) {
+				return `ERR: \`${filename}\` should be imported by \`${correspondingTsxFile}\``;
+			}
+
+			if (entryPointSource.includes(`import './features/${filename}';`)) {
```

Comment:
Can you extract this into a Boolean since it's repeated below?

For example on line 19 add:

```js
const isImportedByEntrypoint = etc
```

Reply:
Addressed in 68bcd6ef6de3c342861027b56fc618d593342de4